scripts/partialBuild.js

```diff
@@ -12,7 +12,8 @@ rollupConfig.plugins.push(partialBuildPlugin);
 rollup.rollup(rollupConfig).then(function(bundle) {
   return bundle.generate(rollupConfig.output);
 }).then(function(result) {
-  process.stdout.write(result.code);
+  var chunk = result.output[0];
+  process.stdout.write(chunk.code);
```

Comment:
Will there always be exactly one chunk? Would it be better to iterate through the chunks calling `write` for each one?

Reply:
I suspect in this case it will always output one chunk, but iterating is definitely better. Changed